test/fixtures/comments.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  image: one
+  user: one
+  text: First post!!!1!
+
+two:
+  image: two
+  user: one
+  text: I like comments
```

Comment:
Maybe we should fixture multiple comments for an image? This seems like something we'll want to test later (e.g. they are sorted in the right order).